lib/gamnit/bmfont.nit

```diff
@@ -406,8 +415,58 @@ class BMFontAsset
 					else 16.0
 				dx += space_advance * scale
 				word_break = true
+			else if c == '[' then
+				# Open link?
+				if i + 1 < text.length and text[i+1] == '[' then
+					# Escape if duplicated
```

Comment:
It would have worked. I preferred `[[` because it is less confusing than having to escape the escape character `\\[`, and it appeared easier to implement in the current code structure.

If this feature turns out to be useful and more text formatting is needed, I'm considering using the markdown parser. Then `TextSprites` would have more standard behaviors.